R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -15,40 +15,35 @@ SparkR is an R package that provides a light-weight frontend to use Apache Spark
 
 ## Getting Started
 
-We begin with an example running on the local machine, trying to provide an overview of the use of SparkR: data ingestion, data processing and machine learning. 
+We begin with an example running on the local machine and provide an overview of the use of SparkR: data ingestion, data processing and machine learning.
 
 First, let's load and attach the package.
 ```{r, message=FALSE}
 library(SparkR)
 ```
 
 
-To use SparkR, you need an Apache Spark package where backend codes to be called are compiled and packaged. 
+To use SparkR, you need an Apache Spark installation which will execute SparkR programs.
```

Comment:
Yeah - I was thinking that we don't want to confuse them too much about SparkR vs. Spark etc. I added back a line about why they need the Spark installation